code/src/main/webapp/JavaScript/script.js

```diff
@@ -187,8 +187,12 @@ function attachSignin(element, negativeUser) {
     var servlet = "";
     if (negativeUser) {
       servlet = '/get-negative-user-info';
+      params.append('calendar', true);
+      params.append('contacts', false);
     } else {
       servlet = '/get-positive-user-info';
+      params.append('calendar', document.getElementById('calendar').checked);
```

Comment:
I think I'd prefer if we attached a list of AuthorizationScopes to the request, rather than individual parameters.
So for example, you'd create a list and then if each of them were checked you would add it to the list. The keywords for this could be the enum names of the AuthorizationScopes in our backend.